docs/src/api/class-frame.md

```diff
@@ -1345,6 +1362,7 @@ Time to wait between `keydown` and `keyup` in milliseconds. Defaults to 0.
 
 ## async method: Frame.querySelector
 * since: v1.9
+* discouraged: Use locator-based [`method: Locator.locator`] instead.
```

Comment:
I think `Frame.locator` is a more suitable replacement. WDYT?

Reply:
Yes, that's what I did in Page, should be same here!